### PR TITLE
CB-21 move to compose v3 and integrate with docker for mac

### PR DESCRIPTION
**Pull request by sodre90**

### CODE REVIEW

File: include/cloudbreak.bash [hunk 1]
```diff
@@ -1,9 +1,12 @@
 
 cloudbreak-config() {
-  : ${BRIDGE_IP:=$(docker run --rm --name=cbreak_cbd_bridgeip --label cbreak.sidekick=true alpine sh -c 'ip ro | grep default | cut -d" " -f 3')}
-  env-import PRIVATE_IP $BRIDGE_IP
-  env-import DOCKER_MACHINE ""
-  compose-config
+  if is_macos; then
+    : ${BRIDGE_ADDRESS:=host.docker.internal}
+    env-import PRIVATE_IP 127.0.0.1
```
Comment (keyki) [inline include/cloudbreak.bash]: we don' need the PRIVATE_IP anymore, or do we?

Comment (sodre90) [reply include/cloudbreak.bash]: we are using only in uaa.yaml, I will try to remove and see what happens

Comment (sodre90) [reply include/cloudbreak.bash]: it looks fine too after I removed PRIVATE_IP

Comment (keyki) [reply include/cloudbreak.bash]: I think Periscope is the one that uses it. Worth trying if it still work in case you run it from idea.

**Comment by keyki** [comment]
Works on my machine

**Comment by keyki** [comment]
DOCKER_TAG_CONSUL can be removed

**Comment by keyki** [comment]
Strange, on VPN cbd start and cbd kill does not work. It just hangs.

**Pull request merged**